preview/Win7Msix/Win7MSIXInstaller/InstallUI.hpp

```diff
@@ -28,13 +28,15 @@ class UI
 public:
     HRESULT ShowUI();
     HRESULT LaunchInstalledApp();
-    void ConfirmAppCancel();
+    void ConfirmAppCancel(HWND parentHWnd);
+    std::wstring g_installOrUpdateText = GetStringResource(IDS_STRING_INSTALLTEXT); /// Default button and install screen UI text is 'Install'

```

Comment:
>g_installOrUpdateText [](start = 17, length = 21)

does this need to be public? also non-global shouldn't start with g_

Reply:
Updated.